### PR TITLE
Update notebook milestones

### PR DESCRIPTION
**Pull request by mjbvz**

### CODE REVIEW

File: .vscode/notebooks/endgame.github-issues [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [
   {
-    "kind": 1,
+    "kind": 1,October 2023
```
Comment (rzhao271) [inline .vscode/notebooks/endgame.github-issues:3]: Is this intentional?

**Review by rzhao271 — CHANGES_REQUESTED** [review]
Invalid diffs.

File: .vscode/notebooks/endgame.github-issues (hunk 1 above)
Comment (mjbvz) [reply .vscode/notebooks/endgame.github-issues:3]: Yeah our find replace support seems to have really messed something up here. Will see if I can create an issue

Comment (mjbvz) [reply .vscode/notebooks/endgame.github-issues:3]: Opened #194632 for replace issue. Manually doing replacements now

**Review by rzhao271 — APPROVED** [review]

**Pull request merged**